### PR TITLE
Add default template

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Set default template implicitly by default

### CODE REVIEW

File: cmd/exo/cmd/config.go [hunk 1]
```diff
@@ -188,6 +188,33 @@ Let's start over.
...
+		}
+		account.DefaultTemplate, err = readInput(reader, "Choose default template", "Linux Ubuntu 18.04 LTS 64-bit")
+		if err != nil {
+			return nil, err
+		}
+		_, err = getTemplateByName(zoneID, account.DefaultTemplate)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, "failed to verify this template") // nolint: errcheck
+		}
+	}
```
Comment (greut) [inline cmd/exo/cmd/config.go]: no needs for that.

File: cmd/exo/cmd/config.go [hunk 2]
```diff
@@ -395,6 +424,33 @@ func importCloudstackINI(option, csPath, cfgPath string) error {
...
+			if err != nil {
+				return err
+			}
+			_, err = getTemplateByName(zoneID, csAccount.DefaultTemplate)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, "failed to verify this template") // nolint: errcheck
+			}
+		}
+
+		cs = nil
```
Comment (greut) [inline cmd/exo/cmd/config.go]: ditto, no needs for that.

File: cmd/exo/cmd/config.go (hunk 1 above)
Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/config.go]: ok

File: cmd/exo/cmd/config.go (hunk 2 above)
Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/config.go]: ok

**Comment by greut** [comment]
do way less. If you have a `defaultTemplate` in the config, take it otherwise use `exo`'s

**Comment by greut** [comment]
```
% exo config show                       
Name: yoan.blanc@exoscale.ch
API Key: EXOc94bc655ff901b7218a5c3cd
API Secret: ×××××××××××××××××××××××××××××××××××××××××××
Account: yoan.blanc@exoscale.ch
Default zone: ch-gva-2
Default template: Debian 9

% exo vm create foo --ipv6

1. Connect to the machine

> exo ssh foo
ssh -i "/home/yoan/.exoscale/instances/c59490b4-24ac-4417-a135-66bdc264c945/id_rsa" debian@2a04:c43:c00:a07:44d:f2ff:fe00:85
```

File: cmd/exo/cmd/vm_create.go [hunk 3]
```diff
@@ -302,7 +306,7 @@ func saveKeyPair(keyPairs *egoscale.SSHKeyPair, vmID string) {
 func init() {
 	vmCreateCmd.Flags().StringP("cloud-init-file", "f", "", "Deploy instance with a cloud-init file")
 	vmCreateCmd.Flags().StringP("zone", "z", "", "<zone name | id | keyword> (ch-dk-2|ch-gva-2|at-vie-1|de-fra-1)")
-	vmCreateCmd.Flags().StringP("template", "t", "Linux Ubuntu 18.04", "<template name | id>")
+	vmCreateCmd.Flags().StringP("template", "t", "", fmt.Sprintf("<template name | id> (default: %s)", defaultTemplate))
```
Comment (pierre-emmanuelJ) [inline cmd/exo/cmd/vm_create.go]: It's wrong it's always be default const value and config file value ! cobra is executed before config 😉

Comment (greut) [reply cmd/exo/cmd/vm_create.go]: I guess I fixed it in the meantime.

Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/vm_create.go]: try and you will see, when you do `exo vm create -h` Cobra execute cmd tree before execute root.go config code

Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/vm_create.go]: I tried your code an hour ago

Comment (greut) [reply cmd/exo/cmd/vm_create.go]: big `T` :)

**Pull request merged**